researcher-reports/src/utils/results-utils.ts

```diff
@@ -62,7 +62,7 @@ export const listResources = async (firebaseJwt: string, amOwner: boolean,
   const client = new TokenServiceClient({ jwt: firebaseJwt, env: tokenServiceEnv });
   return client.listResources({
     type: resourceType,
-    tool: "example-app", // TODO: this needs to be set to valid string based on the app that created the resource
+    tool: "researcher-report",
```

Comment:
Is there any additional work that needs to be done to use this new tool string?  Is it also configured in Firebase?

Reply:
Yes, I created a resource setting in the token-service-staging firestore database: https://console.firebase.google.com/project/token-service-staging/firestore/data~2Fstaging:resourceSettings~2FmIVQo4W7ZwjGr0v4in8W

The only other thing was setting "researcher-report" to be the tool id in the researcher-reports app, which is part of this PR.

Reply:
Great!